function/datastore.js

```diff
@@ -5,7 +5,7 @@ const { rejection } = require('./responseError');
 // Read the project ID from environment
 const projectId = process.env.GCP_PROJECT;
 // Create datastore client
-const Datastore = require('@google-cloud/datastore');
+const {Datastore} = require('@google-cloud/datastore');
```

Comment:
this change required as of `@google-cloud/datastore@3.0.0`: https://github.com/googleapis/nodejs-datastore/releases/tag/v3.0.0